### ISSUE TITLE
Test for Classification on MNIST and Ubran sounds datatset

### ISSUE THREAD

**Issue by wajihullahbaig** [zaydzuhri/softpick-attention#1]
Hi
Thank you for the research publication. 
Another test was run through basic classification problem. It  turns out that softpick is numerically very stable. 
Wanted to ensure its capabilities. 
Results are here. 
https://github.com/wajihullahbaig/audio-classification

Thanks.

**Comment by zaydzuhri** [comment]
Hi! Sorry for the late reply, I never check my github notifcations.

If I'm understanding correctly, you're using softpick in the classifier output? Looks like you got some pretty nice results on the Urban8K dataset. Did you do anything special other than softpick? Like a different loss function, or just CrossEntropyLoss? Why do you think it's working that well? I am curious.

**Comment by wajihullahbaig** [comment]
> Hi! Sorry for the late reply, I never check my github notifcations.
> 
> If I'm understanding correctly, you're using softpick in the classifier output? Looks like you got some pretty nice results on the Urban8K dataset. Did you do anything special other than softpick? Like a different loss function, or just CrossEntropyLoss? Why do you think it's working that well? I am curious.

No worries for the late reply. Using cross entropy as a loss function. Just replaced softmax with softpick. I liked the fact the jerkiness tends to subside.

Good work  by you guys.

Thanks
